packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,215 @@
+# Alert
+
+Alerts are messages that communicate information to users about any significant changes or explanations inside the system in a **prominent way**.
+
+This document outlines the API of `Alert` component.
+
+<img src="./alerts-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Alerts](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6824%3A0)
+
+> **Note**
+>
+> The following needs work:
+>
```

Comment:
These all are done from the Design side. Can you please re-check it once and update it?

Reply:
@kamleshchandnani @divyanshu013 Shall we remove links inside the description for now? Until we have a live use case?

Reply:
Thanks, updated the list. Only one remains now:

> the borderless intent mentioned in full bleed layout shouldn't have any borders and border radii

Reply:
@divyanshu013 This is also done. Please check.

> the borderless intent mentioned in full bleed layout shouldn't have any borders and border radii

Reply:
I think these ones are remaining:

<img width="236" alt="image" src="https://user-images.githubusercontent.com/6682655/182839036-ff5fe3eb-9932-46ea-a18c-2076ffcf8209.png">
 
<img width="222" alt="image" src="https://user-images.githubusercontent.com/6682655/182839079-0d33be1c-c405-4efa-9be5-0962922f511e.png">

Reply:
@divyanshu013 Thanks for pointing it out. Done!